streamly-bytestring.cabal

```diff
@@ -32,7 +32,7 @@ library
   build-depends:
       base >=4.7 && <5
     , bytestring >=0.10.0 && <=0.11.3.1
-    , streamly == 0.8.1.* || == 0.8.2.*
+    , streamly >=0.8.1 && <0.9
```

Comment:
Can you list all the exact versions as this package depends on the internals of streamly. Allowing the entire minor range might break the package for some minor releases.

Reply:
Makes sense, fixed !